### PR TITLE
Detection of availabilty of the MU_SPARQL_ENDPOINT to start querying it

### PR DESCRIPTION
**Pull request by fr0gs**

### CODE REVIEW

**Comment by fr0gs** [comment]
It does not work. It already has depends_on, but that only checks whether
the container has started and not if it is ready to be queried.
Hence the silly healthcheck to see if the endpoint is up.

**Comment by cecton** [comment]
Sorry thye link was vague. I mean did you try the scripts?

> Use a tool such as wait-for-it, dockerize or sh-compatible wait-for. These are small wrapper scripts which you can include in your application’s image and will poll a given host and port until it’s accepting TCP connections.

**Comment by fr0gs** [comment]
Lol no but it does exactly the same. If you prefer to have that logic separated I understand. I will test them.

**Comment by cecton** [comment]
@fr0gs okay I understand. Can you just add requests to the requirements file? Also please test it manually with the changes you need on Virtuoso because the tests are not covering this part.

File: requirements.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ aiohttp < 2.3
 aiosparql < 0.11
 python-dateutil == 2.6.0
 docker-compose == 1.16.0
+requests
```
Comment (cecton) [inline requirements.txt]: @fr0gs can you fix this version?

**Comment by fr0gs** [comment]
Can you merge the code please?

**Pull request merged**